baseline.md

```diff
@@ -0,0 +1,1837 @@
+# Open Source Project Security Baseline
```

Comment:
This file should not have been checked in.

Reply:
fixed!